.github/workflows/ci.yml

```
@@ -205,21 +191,19 @@ jobs:
         run: |
+          export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
           pnpm install --frozen-lockfile
           pnpm lint:check
 
-      # Additional quality checks can be added here
-      # - SonarCloud, CodeQL, etc.
-
   # Summary job for branch protection
   summary:
-    needs: [ build ]
+    needs: [ build-and-test ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)